src/components/search/SearchForm.tsx

```diff
@@ -33,6 +32,15 @@ const InputWrapPostalCode = styled(InputWrap)`
   }
 `
 
+const StyledTextField = styled(TextField)`
+  height: 44px;
+  width: calc(100% - 4px);
+`
+const StyledNumberField = styled(NumberField)`
+  height: 44px;
```

Comment:
`height` can be removed. Is coming from ASC now

Reply:
Done